src/server/app.js

```diff
@@ -1,13 +1,36 @@
-var express = require('express');
-var app = express();
+/* 
 
+	app.js
+
+	Initializes server. Sets up static content. Establishes routes and 
+	their handlers.
+
+	Written by Sam Reaves
+	July 22, 2015
+
+ */
```

Comment:
I like this nice comment header! Let's make it in jsdoc syntax though so we can auto convert it into documentation: 
http://usejsdoc.org/
so maybe something like:
```
/**
 * @name NodeMeme Server
 * @file app.js
 * @description Initializes server. Sets up static content. Establishes routes and their handlers.
 * @author Sam Reaves
 * @date July 22, 2015
 */
```
Ooooh and I just saw this, which might come in handy soon too:
http://usejsdoc.org/howto-commonjs-modules.html